Documentation/Sprint 3/reflection.tex

```diff
@@ -0,0 +1,158 @@
+\item Jaap Heijligers
+\item Jaap Heijligers
+\end{enumerate}
+
+& 
+\begin{enumerate}
+\item 6 hours
+\item 2 hours
+\item 2 hours
+\item 2 hours
+\end{enumerate}
+
+
+&
+\begin{enumerate}
+\item 6 hours
+\item 2 hours
+\item 2 hours
+\item 2 hours
+\end{enumerate}
+&
+\begin{enumerate}
+\item Yes
+\item Yes
+\item Yes
+\item Yes
+\end{enumerate}
+&
+\begin{enumerate}
+\item 
+\item 
+\item 
+\item JW was in charge of the changes. This has been an oversight in the product planning.
+\end{enumerate}
+\\
+\end{tabular}
+\end{table}
+\begin{table}[h]
+\begin{tabular}{|p{3cm}|p{5.0cm}|p{3.0cm}|p{2.2cm}|p{2.2cm}|p{2.0cm}|p{5cm}}
+As a Scrum Master, I want my team to have a clear vision of the architecture of the system we will be building this sprint.
+&
+\begin{enumerate}
+\item Brainstorm about best architecture
+\item Prototype architectures
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item Gijs Weterings
+\item Tim van der Lippe
+\end{enumerate}
+
+&
+
+\begin{enumerate}
+\item 4 hours
+\item 4 hours
+\end{enumerate}
+&
+\begin{enumerate}
+\item 26 hours
+\item 0 hours
+\end{enumerate}
+&
+\begin{enumerate}
+\item Yes
+\item No
+
+\end{enumerate}
+&
+\begin{enumerate}
+\item This issue has taken up a lot of time. This is partly due to the fact we only guessed 4 hours, this should have been 4 hours per person. Also, we took the time because we have made some pretty big changes to our future planning. We will deviate from the neural network we planned, because getting enough training data would be a master thesis size project, according to Cynthia. We had a meeting with Cynthia on Tuesday, to strategize our planning. We agreed on dropping the neural network, and search for a simpler way of getting past a cold start. Most of the training will be delegated to users, possibly in a gamification manner.
+\item Due to the big changes in the issue above, we opted to delay the prototyping to a point where we have a better strategy.
+\end{enumerate}
+\end{tabular}
+\end{table}
+\begin{table}[h]
+\begin{tabular}{|p{3cm}|p{5.0cm}|p{3.0cm}|p{2.2cm}|p{2.2cm}|p{2.0cm}|p{5cm}}
+As a developer I want my frontend to have a more interactive communication with the backend.
+&
+\begin{enumerate}
+\item Get a clear definition of the responses for the API calls.
+\item Make a call to the API to get a list of rooms based on selected mood.
+\item Make a call to the API to retrieve new messages and insert them in the chatbox.
+\end{enumerate}
+
+&
+
+\begin{enumerate}
+\item Jaap Heijligers
+\item Gijs Weterings
+\item Eva Anker
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item 6 hours
+\item 2 hours
+\item 2 hours
+\end{enumerate}
+&
+\begin{enumerate}
+\item 10 hours
+\item 2 hours
+\item 2 hours
+\end{enumerate}
+&
+\begin{enumerate}
+\item Yes
+\item Yes
+\item Yes
+\end{enumerate}
+&
+\begin{enumerate}
+\item We opted to use the time we had left to get a clear view of the API work. We held this conversation with 4 of us, which is why we have extra time here.
```

Comment:
I don't think we can state this already as the meeting is still planned. Probably 2-3 persons are enough to design the API calls.